web/styles/legacy_more_dense_mode.css

```diff
@@ -0,0 +1,10 @@
+.more-dense-mode {
+    .rendered_markdown .emoji {
+        /* "Eyeballed" styles to compensate for inline-block emoji
+          positioning in messages where the height of the emoji
+          exceeds the line-height in the message area. */
+        position: relative;
+        margin-top: -7px;
+        top: 3px;
+    }
```

Comment:
I wonder if if we could replace this particular set of hacky positioning code with something to overflow the container that looks more like this:
```
padding-top: -1px;
padding-bottom: -1px;
height: calc(var(--line-height-fitted-length-px) + 2px);
width: calc(var(--line-height-fitted-length-px) + 2px);
```

The potential benefit being that it'd be less finicky than the `-7` / `3` hacky numbers? Not sure, just brainstorming.

Reply:
Thanks for the feedback, Tim. I've left this PR in situ and opened #30050 to work through a different approach [as suggested on CZO](https://chat.zulip.org/#narrow/stream/101-design/topic/info.20density.3A.20inline.20emoji.20presentation/near/1796448). I apologize for not giving you a heads-up on that, but this feedback might still be useful applied to that PR.